pycon-us-2018/videos/lightning-talks-friday-pycon-2018.json

```diff
@@ -0,0 +1,103 @@
+{
+  "copyright_text": "Standard YouTube License",
+  "description": ".. list-table:: Lightning Talks\n   :widths: 10 30 60\n   :header-rows: 1\n\n\n   * - Time\n     - Speaker\n     - Title\n   * - 00:40_\n     - Katie Cunningham\n     - You need a hobby\n   * - 05:01_\n     - \u0141ukasz Langa\n     - Black\n   * - 10:05_\n     - Sage Sharp\n     - Outreachy\n   * - 14:44_\n     - Joshua Lowe\n     - Edublocks\n   * - 18:07_\n     - Mahmoud Hasemi\n     - Pycon 2018\n   * - 23:00_\n     - Quang Wu\n     - Python code style for blind programmers\n   * - 28:50_\n     - Alex Rosengarten\n     - Music from chaos: Audiofying the Lorenz Attractor\n   * - 34:05_\n     - Sahuna Gordon-McKeon\n     - Legacies of disruption\n   * - 39:02_\n     - Travis Oliphant\n     - More sustainable Open Source!\n   * - 42:07_\n     - James Powell\n     - Code or die\n   * - 48:56_\n     - Scott Shawcroft\n     - Circuit Python\n   * - 51:57_\n     - Sean Harrington\n     - Hacking in 23&Me and DbSNP\n\n\n.. _00:40: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=00m40s\n.. _05:01: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=05m01s\n.. _10:05: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=10m05s\n.. _14:44: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=14m44s\n.. _18:07: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=18m07s\n.. _23:00: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=23m00s\n.. _28:50: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=28m50s\n.. _34:05: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=34m05s\n.. _39:02: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=39m02s\n.. _42:07: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=42m07s\n.. _48:56: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=48m56s\n.. _51:57: https://www.youtube.com/watch?v=bTAFl8P2DkE&t=51m57s\n",
```

Comment:
Mahmoud Hasemi's talk title is incorrect in the description here, it should be "Restructuring Data in Python"

Reply:
And one more comment for this line, this one's just a typo: "Hacking in 23&Me and DbSNP" should be "Hacking on 23&Me and DbSNP"

Reply:
Mahmoud Hasemi's talk title should be a "copy" and "paste" bug :sweat_smile:
"Hacking on 23&Me and DbSNP" is totally on me :innocent: 

(And I forgot to add the speakers)